doc/application_plugins/openvpn.rst

```diff
@@ -26,6 +26,19 @@ The entered password is sent to privacyIDEA. Thus you can require the user to
 enter a password consisting of a static part he knows and the OTP part which
 the user needs to generate with the OTP token he possesses.
 
+Another addition you most probably want to make is adding the following option
+to both the client and the server configuration:
+
+   reneg-sec 0
+
+By default, the channel key gets renegotiated after 3600 seconds, either
+partner can request a renegotiation. If only one partner disables this
+option, the other one will request it.
+
+When using dual-factor authentication, note that this default 
```

Comment:
Please change this to "one time password authentication" or something similar. two factor authentication is nowadays often confused with OTP. But OTP is only one possiblity of 2FA. 
You could very well do two factor authentication with smartcards without any problem!